.eslintrc.json

```diff
@@ -125,18 +126,19 @@
       "files": ["*.ts"],
       "parser": "@typescript-eslint/parser",
       "plugins": [
-        "@typescript-eslint",
-        "eslint-plugin-tsdoc"
+        "@typescript-eslint"
       ],
       "extends": ["airbnb-typescript/lib/shared"],
       "rules": {
         "@typescript-eslint/no-implied-eval": "off",
         "@typescript-eslint/no-throw-literal": "off",
         "comma-dangle": ["warn", "never"],
+        "import/first": "off",
+        "import/prefer-default-export": "off",
         "lines-between-class-members": "off",
+        "lodash/prefer-lodash-typecheck": "off",
```

Comment:
If we're going to require lodash everywhere, don't we want to encourage its use?

Reply:
We aren't requiring `lodash`, but we are using it when it's available and when it makes sense. This rule was forcing us to do things like `isUndefined(x)` instead of `x === undefined`, which isn't necessary.